packages/@webex/internal-plugin-scheduler/test/unit/spec/scheduler/scheduler.js

```diff
@@ -1,8 +1,10 @@
-import {assert} from '@webex/test-helper-chai';
 import MockWebex from '@webex/test-helper-mock-webex';
 
 import Scheduler from '@webex/internal-plugin-scheduler';
 
+import CONSTANTS from '@webex/internal-plugin-scheduler/src/scheduler/scheduler.constants';
```

Comment:
Assuming integration tests are not possible at this time due to the overlap of `internal-plugin-calendar`?